ironfish-cli/src/commands/wallet/notes/index.ts

```diff
@@ -33,74 +33,74 @@ export class NotesCommand extends IronfishCommand {
     const account = await useAccount(client, flags.account)
 
     const response = client.wallet.getAccountNotesStream({ account })
-
-    let showHeader = !flags['no-header']
-
+    const notes: RpcWalletNote[] = []
     for await (const note of response.contentStream()) {
+      if (notes.length >= flags.limit) {
+        break
+      }
       if (!assetLookup.has(note.assetId)) {
         assetLookup.set(
           note.assetId,
           (await client.wallet.getAsset({ id: note.assetId, account })).content,
         )
       }
```

Comment:
These were rendering entire tables as individual rows. I updated to collect from the stream, not sure if this is desirable as at very large number of rows could cause memory issue (also output will not stream), but seemed like a reasonable tradeoff.